build/ToolsetPackages/RoslynToolset.csproj

```diff
@@ -31,7 +31,9 @@
     <PackageReference Include="RoslynTools.Microsoft.LocateVS" Version="$(RoslynToolsMicrosoftLocateVSVersion)" />
     <PackageReference Include="RoslynTools.Microsoft.SignTool" Version="$(RoslynToolsMicrosoftSignToolVersion)" />
     <PackageReference Include="RoslynTools.Microsoft.VSIXExpInstaller" Version="$(RoslynToolsMicrosoftVSIXExpInstallerVersion)" />
+    <PackageReference Include="RoslynTools.MSBuild" Version="$(RoslynToolsMSBuildVersion)" />
     <PackageReference Include="xunit.runner.wpf" Version="$(xunitrunnerwpfVersion)" />
+    <PackageReference Include="vswhere" Version="$(vswhereVersion)" />
```

Comment:
Why do we need RoslynTool.MSBuild and vswhere?

Reply:
RoslynTool.MSBuild: this is our xcopy msbuild package 
vswhere: helps us avoid downloading the xcopy msbuild package by using msbuild installed on machine if it's of a sufficient version.